cmd/helm/inspect.go

```diff
@@ -82,11 +86,37 @@ func newInspectCmd(out io.Writer) *cobra.Command {
 		output: all,
 	}
 
+	readPassword := func() error {
```

Comment:
could we refactor all this into a general function? Something like

```
func readPassword(username, password string, readPasswordFromStdin bool) (string, error) {
}
```

That way we're not duplicating the same code in 5 different places, and the code in each command can just be

```
password, err := readPassword(insp.username, insp.password, insp.passwordStdin)
if err != nil {
    return err
}
```

Which makes maintenance for this particular feature *much* more manageable.

Reply:
Definitely; I was thinking along similar lines but wasn't sure where to put the new function -- currently leaning towards a new file. I'll think of a first draft location and will probably make it take a `Reader` as well so that I can write some unit tests for it.

Reply:
In general we've been placing helper functions under `cmd/helm/helm.go` but if it makes sense to go under `pkg/` then perhaps creating a new package called `pkg/password`? Up to you.

Reply:
OK, thanks! I'll likely keep it internal to start, so `cmd/helm/helm.go` seems like the preferred choice at the moment. I'll take a shot at this later today.

Reply:
Update: I did make an update but have forgotten the magic I needed to do in order to do a build; the `glide` command is running out of file descriptors. I remember it was something to do with `docker --ulimit nofiles=1000000` but still ran into an error. Will try building outside a container tomorrow.

Also: Is there an explicit reason why the `helm inspect` command and its children didn't use `PersistentFlags`? There seems to be a bunch of code that could be eliminated by using it. Perhaps I'll find out when I get the build working...